package.json

```diff
@@ -28,6 +28,12 @@
     "test:karma": "cross-env NODE_ENV=test karma start test/karma.conf.js",
     "test:unit": "cross-env NODE_ENV=test mocha 'packages/**/*.test.tsx' --exclude '**/node_modules/**'",
     "test:watch": "yarn test:unit --watch",
+    "test:regressions": "cross-env NODE_ENV=production yarn test:regressions:build && concurrently --success first --kill-others \"yarn test:regressions:run\" \"yarn test:regressions:server\"",
+    "test:regressions:build": "webpack --config test/regressions/webpack.config.js",
+    "test:regressions:dev": "concurrently \"yarn test:regressions:build --watch\" \"yarn test:regressions:server\"",
+    "test:regressions:run": "mocha --config test/regressions/.mocharc.js --delay 'test/regressions/**/*.test.js'",
+    "test:regressions:server": "serve test/regressions",
+    "test:argos": "node ./scripts/pushArgos.js",
```

Comment:
Isn't there a way to reduce the number of scripts?

Reply:
You only have to run one script `yarn test:regressions` locally. The others are here to help debug (when working on improving the regression generation tool.